.vscode/solution-explorer/class.ts-template

```diff
@@ -0,0 +1,3 @@
+export class {{name}} {
```

Comment:
IMHO I wouldn't add any of these vscode templates in this PR. It's already pretty big and hard to review.
It'd be ideal to get these in (if at all) in a separate PR.